app/controllers/api/providers_controller.rb

```diff
@@ -52,6 +53,18 @@ def delete_resource(type, id = nil, _data = nil)
 
     private
 
+    def update_custom_attributes(provider, custom_attributes)
```

Comment:
I see you are using this method below - is it intentionally patching the method defined in `Subcollections::CustomATtributes`?